File: C++/2850.cpp
#include <iostream>
#include <cmath>
using namespace std;

int contaDig(long long valor);

int main(){
    string perna;
    while(getline(cin,perna)){
        if(perna == "esquerda"){
            cout << "ingles" << endl;
        }
        else if(perna == "direita"){
            cout << "frances" << endl;
        }
        else if(perna == "nenhuma"){
            cout << "portugues" << endl;
        }
        else{
            cout << "caiu" << endl;
        }
    }

    return 0;
}

File: Go/2850.go
package main

import (
    "bufio"
    "fmt"
    "os"
)

func main () {
    var str string
    scanner := bufio.NewScanner(os.Stdin)

    for scanner.Scan() {
        str = scanner.Text()
        if(str == "esquerda"){
            fmt.Println("ingles")
        } else if(str == "direita"){
            fmt.Println("frances")
        } else if(str == "nenhuma"){
            fmt.Println("portugues")
        } else{
            fmt.Println("caiu")
        }
    }
}
